Four use cases were unsuccessful in executing the test.

_Please use the following template to help us solving your issue._

## Issue Details
hi，four use cases were unsuccessful in executing the test.
 ``` 
  1629 - execution___of__Point_WKT (Subprocess aborted)			
  1637 - execution___of__read_WKT (Subprocess aborted)
  1644 - compilation_of__off2off (Failed)
  1645 - execution___of__off2off (Not Run)
```
The detailed error report is roughly as follows：
```
1629/2025 Testing: execution___of__Point_WKT
1629/2025 Test: execution___of__Point_WKT
Command: "/home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/build/examples/Stream_support/Point_WKT"
Directory: /home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/build/examples/Stream_support/__exec_test_dir
"execution___of__Point_WKT" start time: Apr 15 11:08 CST
Output:
----------------------------------------------------------
error.
terminate called after throwing an instance of 'CGAL::Assertion_exception'
  what():  CGAL ERROR: assertion violation!
Expr: (!is_valid(i)) || (!is_valid(s)) || (!(i>s))
File: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/Interval_nt.h
Line: 161
Explanation:  Variable used before being initialized (or CGAL bug)
<end of output>
Test time =   0.01 sec
----------------------------------------------------------
Test Failed.
"execution___of__Point_WKT" end time: Apr 15 11:08 CST
"execution___of__Point_WKT" time elapsed: 00:00:00
----------------------------------------------------------


test 1644
    Start 1644: compilation_of__off2off

1644: Test command: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cmake-3.19.5-ua3ox4pbrahwkz47obagiaepfsn4blmn/bin/cmake "--build" "/home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/build" "--target" "off2off" "--config" ""
1644: Test timeout computed to be: 10000000
1644: Building CXX object examples/Stream_support/CMakeFiles/off2off.dir/off2off.cpp.o
1644: In file included from /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/Scanner_OFF.h:22:0,
1644:                  from /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF.h:14,
1644:                  from /home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/examples/Stream_support/off2off.cpp:4:
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:37:8: error: 'vector' in namespace 'std' does not name a template type
1644:    std::vector<double> entries;
1644:         ^~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_vertex(double&, double&, double&, double&)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:91:7: error: 'entries' was not declared in this scope
1644:        entries.clear();
1644:        ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_texture(double&, double&, double&)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:212:13: error: 'entries' was not declared in this scope
1644:          x = entries[first_texture_index];
1644:              ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_normal(double&, double&, double&, double&)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:303:12: error: 'entries' was not declared in this scope
1644:          if(entries.size() <= first_normal_index + 2)
1644:             ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:310:13: error: 'entries' was not declared in this scope
1644:          x = entries[first_normal_index];
1644:              ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_color(unsigned char&, unsigned char&, unsigned char&)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:612:52: error: 'entries' was not declared in this scope
1644:          color = get_indexed_color(static_cast<int>(entries[first_color_index])); // the index in the color map
1644:                                                     ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:618:19: error: 'entries' was not declared in this scope
1644:        double rd = entries[first_color_index];
1644:                    ^~~~~~~
1644: In file included from /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/Scanner_OFF.h:22:0,
1644:                  from /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF.h:14,
1644:                  from /home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/examples/Stream_support/off2off.cpp:4:
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_facet(std::size_t&, std::size_t)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:705:7: error: 'entries' was not declared in this scope
1644:        entries.clear();
1644:        ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h: In member function 'void CGAL::File_scanner_OFF::scan_facet_vertex_index(std::size_t&, const size_t&, std::size_t)':
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:740:10: error: 'entries' was not declared in this scope
1644:        if(entries.size() <= current_entry )
1644:           ^~~~~~~
1644: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/include/CGAL/IO/OFF/File_scanner_OFF.h:748:40: error: 'entries' was not declared in this scope
1644:        index = static_cast<std::size_t>(entries[current_entry]);
1644:                                         ^~~~~~~
1644: examples/Stream_support/CMakeFiles/off2off.dir/build.make:81: recipe for target 'examples/Stream_support/CMakeFiles/off2off.dir/off2off.cpp.o' failed
1644: gmake[3]: *** [examples/Stream_support/CMakeFiles/off2off.dir/off2off.cpp.o] Error 1
1644: CMakeFiles/Makefile2:24828: recipe for target 'examples/Stream_support/CMakeFiles/off2off.dir/all' failed
1644: gmake[2]: *** [examples/Stream_support/CMakeFiles/off2off.dir/all] Error 2
1644: CMakeFiles/Makefile2:24835: recipe for target 'examples/Stream_support/CMakeFiles/off2off.dir/rule' failed
1644: gmake[1]: *** [examples/Stream_support/CMakeFiles/off2off.dir/rule] Error 2
1644: Makefile:9684: recipe for target 'off2off' failed
1644: gmake: *** [off2off] Error 2
3/5 Test #1644: compilation_of__off2off ..................***Failed    3.50 sec
test 1645
    Start 1645: execution___of__off2off
Failed test dependencies: compilation_of__off2off
4/5 Test #1645: execution___of__off2off ..................***Not Run   0.00 sec
test 1631
    Start 1631: Stream_support_Examples_CleanupFixture

1631: Test command: /home/all_spack_env/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cmake-3.19.5-ua3ox4pbrahwkz47obagiaepfsn4blmn/bin/cmake "-E" "remove_directory" "/home/all_spack_env/spack_stage/root/spack-stage-cgal-5.4-nl3m6b3j5yhm4t6zft7xvyvxsnjsevlh/spack-src/build/examples/Stream_support/__exec_test_dir"
1631: Test timeout computed to be: 10000000
5/5 Test #1631: Stream_support_Examples_CleanupFixture ...   Passed    0.01 sec

The following tests passed:
        check_build_system
        Stream_support_Examples_SetupFixture
        Stream_support_Examples_CleanupFixture

60% tests passed, 2 tests failed out of 5

Label Time Summary:
CGAL_build_system          =   0.30 sec*proc (1 test)
Stream_support_Examples    =   3.52 sec*proc (4 tests)

Total Test time (real) =   3.96 sec

The following tests FAILED:
        1644 - compilation_of__off2off (Failed)
        1645 - execution___of__off2off (Not Run)
Errors while running CTest

```

The steps to reproduce are as follows：
```
mkdir build
cd build
cmake -DBUILD_TESTING=ON -DWITH_examples=ON ..
ctest -j96

```

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):linux ubuntu 18.04.5 aarch64
* Compiler: gcc 7.5.0
* Release or debug mode:Release 
* Specific flags used (if any):
* CGAL version:5.4
* Boost version:1.7.6
* Other libraries versions if used (Eigen, TBB, etc.):

--- a/Filtered_kernel/include/CGAL/Lazy.h
+++ b/Filtered_kernel/include/CGAL/Lazy.h
@@ -524,9 +524,7 @@ public:
   mutable std::atomic<ET*> ptr_ { nullptr };
   mutable std::once_flag once;
 
-  Lazy_rep () {}
-
-  Lazy_rep (AT a)
+  Lazy_rep (AT a = AT())
       : x(-a.inf()), y(a.sup()) {}
 
   template<class E>
@@ -535,7 +533,13 @@ public:
 
   AT approx() const
   {
-    return AT(-x.load(std::memory_order_relaxed), y.load(std::memory_order_relaxed));
+    // Do not check that the interval is valid. Indeed, using IO/WKT/traits_point.h for instance,
+    // one can default-construct a point, then set X, and then Y, which amounts to
+    // Point_2(Point_2(X, Point_2().y()).x(), Y).
+    // With Epeck, we have a default constructed array of Interval_nt in Point_2(),
+    // then .y() returns a Lazy_exact_nt containing an invalid interval,
+    // and when we read that interval we end up here.
+    return AT(-x.load(std::memory_order_relaxed), y.load(std::memory_order_relaxed), typename AT::no_check_t());
   }
 
   void set_at(ET*, AT a) const {

--- a/Number_types/include/CGAL/Interval_nt.h
+++ b/Number_types/include/CGAL/Interval_nt.h
@@ -110,6 +110,8 @@ public:
     bool exact = ((unsigned long long)d == i) || (i <= safe);
     if (!CGAL_CST_TRUE(exact))
 #endif
+      // This requires a suitable rounding mode, which we always set for
+      // arithmetic, but not always for a conversion...
       *this += smallest();
   }
 
@@ -146,12 +148,21 @@ public:
   explicit Interval_nt(__m128d v) : val(v) {}
 #endif
 
-  Interval_nt(double i, double s)
+  // Unchecked version for Lazy_rep in Lazy.h.
+  struct no_check_t {};
+  Interval_nt(double i, double s, no_check_t)
 #ifdef CGAL_USE_SSE2
     : val(_mm_setr_pd(-i, s))
 #else
     : _inf(-i), _sup(s)
 #endif
+  {
+#ifndef CGAL_DISABLE_ROUNDING_MATH_CHECK
+    CGAL_assertion_code((void) tester;) // Necessary to trigger a runtime test of rounding modes.
+#endif
+  }
+
+  Interval_nt(double i, double s) : Interval_nt(i, s, no_check_t())
   {
     // Previously it was:
     //    CGAL_assertion_msg(!(i>s);
@@ -159,9 +170,6 @@ public:
     // /fp:strict. If 'i' or 's' is a NaN, that makes a difference.
     CGAL_assertion_msg( (!is_valid(i)) || (!is_valid(s)) || (!(i>s)),
               " Variable used before being initialized (or CGAL bug)");
-#ifndef CGAL_DISABLE_ROUNDING_MATH_CHECK
-    CGAL_assertion_code((void) tester;) // Necessary to trigger a runtime test of rounding modes.
-#endif
   }
 
   Interval_nt(const Pair & p)

